lib/apiservers/engine/backends/container_proxy.go

```diff
@@ -681,6 +683,26 @@ func (c *ContainerProxy) StreamContainerStats(ctx context.Context, config *conve
 	return nil
 }
 
+func (c *ContainerProxy) GetContainerChanges(vc *viccontainer.VicContainer) (io.ReadCloser, error) {
```

Comment:
Can you pass ctx down to this function? That way you can use the same context for the below call (ArchiveExportReader)